front/app/components/ReactionControl/ReactionButton.tsx

```diff
@@ -504,7 +504,9 @@ const ReactionButton = ({
             </Button>
           )}
           {disabledReason && (
-            <ScreenReaderOnly id={describedById}>
+            <ScreenReaderOnly
+              id={disabledReasonMessage ? undefined : describedById}
+            >
```

Comment:
From my understanding here, when there is a `disabledMessage`, that also has an `id={describedById}`, so two elements have the same ID. I've made this one conditional instead, so this id will only ever be used either by this `ScreenReaderOnly` or the `span` in the `disabledMessage` element.

Reply:
This code and its type aren't the greatest, indeed. 

Basically, ScreenReaderOnly will only render if there's a disabledReason and then there will be a disabledReasonMessage, so this will normally always be undefined.

You could remove the span wrapper around the FormattedMessage of disabledMessage or remove it from ScreenReaderOnly. One place where we define id suffices.

Let me know if you want me to make a quick PR. It is often easier to explain something with code than with text. :)

Reply:
Pretty confusing stuff if I look at this code. So I'm not confident about my comments above. What I would recommend is to search for the id in the DOM and see if it's there only 1x. And only define it in one spot (on the span wrapping the disabled message or ScreenReaderOnly, whichever one makes most sense).